packages/clay-shared/src/getEllipsisItems.tsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import React from 'react';
+
+const getBufferList = (
+	items: Array<Object>,
+	start: number,
+	end: number,
+	EllipsisComponent: React.ComponentType<any>,
+	ellipsisProps?: Object,
+	spritemap?: string
+): Array<Object | number | React.ReactElement> => {
+	const removedItems = items.slice(start, Math.max(end, start));
+
+	if (removedItems.length > 1) {
+		return [
```

Comment:
How come we are returning an array here?

Reply:
Since the return must be an array, makes sense encapsulate the component in an array previously.